src/doc/src/reference/cargo-targets.md

```diff
@@ -313,6 +306,13 @@ name = "my-pg-tool"
 required-features = ["postgres", "tools"]
 ```
 
+### The `edition` field
+
+The `edition` field defines the [Rust edition] the target will use. If not
+specified, it defaults to the [`edition` field][package-edition] for the
+`[package]`.
+
+> **Note:* This field is deprecated.
```

Comment:
Do you intentionally write this without saying it will be removed in next edition, and why?

Reply:
In the PR, I had said

> Leaving off any talk of it being removed until we know the edition.

Though I guess we do sometimes say "this will be removed in the future", so maybe that could work.

Reply:
The current one is good enough